sdk/loadtestservice/Azure.Developer.LoadTesting/samples/Sample4_CreateAndUpdateTest.md

```diff
@@ -0,0 +1,47 @@
+# Create And Update Test
+
```

Comment:
How is this different from Create or Update test?

Reply:
This is for test run added naming to doc